moveit_ros/visualization/robot_state_rviz_plugin/include/moveit/robot_state_rviz_plugin/robot_state_display.h

```diff
@@ -132,7 +132,7 @@ private Q_SLOTS:
   robot_state::RobotStatePtr robot_state_;
   std::map<std::string, std_msgs::ColorRGBA> highlights_;
   bool update_state_;
-  bool load_robot_model_;  // for delayed robot initialization
```

Comment:
You need to add:

```c++
#include <atomic>
```

to the top of this file for this to compile on the version of the compiler used for kinetic.

Reply:
done